flink-formats/flink-protobuf/src/test/java/org/apache/flink/formats/protobuf/VeryBigPbProtoToRowTest.java

```diff
@@ -20,7 +20,7 @@
 
 import org.apache.flink.formats.protobuf.testproto.VeryBigPbClass;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
That one looks like it's accidentally added?

Reply:
uuuh, jup.